phoronix/run_phoronix.sh

```diff
@@ -119,16 +119,23 @@ if [ $show_usage -eq 1 ]; then
 fi
 
 test_tools/package_tool --update
-for pkg in cli common php; do
-	package=`dnf list | grep  php  | grep $pkg | awk '{print $1}'`
-	if [[ $package == "" ]]; then
-		error_out "Did not find a php $pkg package" 1
-	fi
-	test_tools/package_tool --packages $package
-	if [[ $? != "0" ]]; then
-		error_out "Failed to install one or more $php_cli,$php_common,$php_xml" 1
-	fi
-done
+case "`test_tools/detect_os`" in
+	"ubuntu")
+		packages="php-cli,php,php-common"
+	;;
+       	*)
+		for pkg in cli common php; do
+			packages=`dnf list | grep  php  | grep $pkg | awk '{print $1}'`
+			if [[ $packages == "" ]]; then
+				error_out "Did not find a php $pkg package" 1
+			fi
+		done
+	;;
+esac
+test_tools/package_tool --packages "php-cli,php,php-common"
+if [[ $? != "0" ]]; then
+	error_out "Failed to install one or more php_cli,php_common,php_xml" 1
+fi
```

Comment:
If you're going to install php-cli,php,php-common, why do you need the `case`?

Also why are we installing packages in the wrapper?  Shouldn't we be using the configuration files in Zathras for this?

Reply:
Think about it, you folks are right, lets move the packages to the configs.  We can deal with the package names changing as need be.